src/renderer/store/modules/utils.js

```diff
@@ -57,6 +58,10 @@ const getters = {
     return state.sessionSearchHistory
   },
 
+  getDeArrowCache () {
```

Comment:
There is no need to return whole cache object, just the entry for video ID is enough

See `getSubscriptionsCacheEntriesForOneChannel` in https://github.com/FreeTubeApp/FreeTube/pull/3668